account_payment_term_extension/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<data>
+
+     <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.account</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//app[@name='account']" position="inside">
+                <block title="Payment Terms" name="payment_terms_setting_container">
+                    <setting
+                        id="payment_terms_delay"
+                        string="Payment Terms Delay Types"
+                        company_dependent="1"
+                        help=""
+                        title="Configure the type of delay for payment terms."
+                    >
```

Comment:
I'd say to add a little description for the purpose of the setting so it is visible for the user. Something like: "Select the Delay Type options that will be available in Payment Terms." What do you think?